publicsuffixlist.js

```diff
@@ -13,8 +13,9 @@
 
 /*! Home: https://github.com/gorhill/publicsuffixlist.js -- GPLv3 APLv2 */
 
-/* jshint browser:true, esversion:6, laxbreak:true, undef:true, unused:true */
-/* globals WebAssembly, console, exports:true, module */
+/* globals WebAssembly, exports:true, module */
```

Comment:
This file is entirely copied from https://github.com/gorhill/uBlock/tree/master/src/lib/publicsuffixlist. There's no change otherwise. The goal is to bring them in sync.

The question of how `enableWASM()` should work still remains. In the worst case, we add some documentation on how to write your own `wasmModuleFetcher()` implementation. Another option would be to have default built-in implementations for both Node.js and HTML.